src/treeChooser/treeChooser.controller.js

```diff
@@ -397,6 +397,27 @@ function TreeChooserController(
   };
 
   /**
+   * Hide all child nodes in a root parents
+   */
+  vm.minimize = function () {
```

Comment:
I think you can just use vm.reset() here.

Reply:
You're right. With vm.reset() it works the same. Cleared.